api/db-cdr/generate-cdr/validate-prerequisites-exist.sh

```diff
@@ -61,14 +70,21 @@ if gsutil -m cp gs://$BUCKET/$BQ_DATASET/$CSV_HOME_DIR/*.csv $TEMP_FILE_DIR
           gsutil cp $TEMP_FILE_DIR/$file.gz gs://$BUCKET/$BQ_DATASET/$CSV_HOME_DIR/backup/"$timestamp"_"$file".gz
         fi
       ;;
-    $PREP_CDR_DATE|$PREP_CRITERIA|$PREP_CRITERIA_ANCESTOR|$PREP_CLINICAL_TERMS|$PREP_CONCEPT|$PREP_CONCEPT_RELATIONSHIP)
+    $CB_SURVEY_VERSION | \
```

Comment:
Not part of this PR but just a note : Maybe we can create a variable like ALL_FILES to store this subset of file names as the list is too long now